core/src/luigi-config/features.js

```diff
@@ -41,3 +44,11 @@ export async function resolveFeatures(features, data) {
   );
   return Object.fromEntries(entries);
 }
+
+export async function getFeatures(data = null) {
+  const rawFeatures =
+    (await getActiveCluster())?.features ||
+    (await getBusolaClusterParams()).config?.features;
```

Comment:
maybe just in case:
```suggestion
    (await getBusolaClusterParams())?.config?.features;
```
?